account_multi_foreign_vat/models/partner.py

```diff
@@ -0,0 +1,30 @@
+from odoo import api, models
+
+
+class AccountFiscalPosition(models.Model):
+
+    _inherit = "account.fiscal.position"
```

Comment:
You are inheriting twice account.fiscal.position in tow different files. Can you move this extension to the file with the name of the model?

Reply:
Sure, my bad